schemas/records.json

```diff
@@ -647,33 +519,39 @@
       "type": "object",
       "properties": {
         "rel": {
-          "type": "string",
-          "description": "The type or semantics of the relation."
+          "title": "Relation",
+          "type": "string"
         },
         "type": {
           "type": "string",
-          "description": "A hint indicating what the media type of the result of dereferencing the link should be."
+          "title": "Media Type"
         },
         "hreflang": {
           "type": "string",
-          "description": "A hint indicating what the language of the result of dereferencing the link should be."
+          "description": "A hint indicating what the language of the result of dereferencing the link should be.",
+          "options": {
+            "hidden": true
+          }
         },
         "title": {
           "type": "string",
-          "description": "Used to label the destination of a link such that it can be used as a human-readable identifier."
+          "title": "Title"
         },
         "length": {
-          "type": "integer"
+          "type": "integer",
+          "options": {
+            "hidden": true
+          }
         },
         "created": {
-          "type": "string",
-          "description": "Date of creation of the resource pointed to by the link.",
-          "format": "date-time"
+          "options": {
+            "hidden": true
+          }
```

Comment:
This removed the validation, why?

Reply:
This is the "created" property of links - do we need that?

Reply:
No, but you removed the data type and format, too. So it's not validated any longer.

Reply:
Ah, sorry, that was a mistake! Thanks for spotting!. Reverted in https://github.com/ESA-EarthCODE/open-science-catalog-validation/pull/30/commits/ef382b3005231f2a6139d9ae95b1ca4aec360c86, also for `updated` property below.